Core/Core/Discussions/DiscussionHTML.swift

```diff
@@ -238,7 +238,8 @@ public enum DiscussionHTML {
             entries.length > 0 && h('h2', { class: \(s(.heading)) },
                 \(s(NSLocalizedString("Replies", bundle: .core, comment: "")))
             ),
-            entries.map(entry => h(Discussion.Entry, { key: entry.id, topic, entry, depth: 0, maxDepth, canLike }))
+            entries.map(entry => h(Discussion.Entry, { key: entry.id, topic, entry, depth: 0, maxDepth, canLike })),
+            h('script', { dangerouslySetInnerHTML: { __html: \(s(CoreWebView.ltiJavaScript)) } })
```

Comment:
Rather than re-include an inline script that gets updated in each render pass, which I don't know if it will re-execute each time, I'd prefer the `CoreWebView`'s js expose a function you can call after `preact.render`.

Reply:
I see what you mean. Tell me if the new changes are what you had in mind.